WORKSPACE

```diff
@@ -1,5 +1,9 @@
 workspace(name = "envoy")
 
+load("//bazel:repo.bzl", "envoy_repo_binding")
+
+envoy_repo_binding()
```

Comment:
These could be rolled into repositories.bzl

Reply:
ill move